test/support/requests_generator.ex

```diff
@@ -0,0 +1,116 @@
+defmodule Xcribe.Support.RequestsGenerator do
+  alias Plug.Conn
+  alias Phoenix.ConnTest
+  alias Xcribe.ConnParser
+
+  require Phoenix.ConnTest
+
+  import Xcribe.WebRouter.Helpers
+
+  @endpoint Xcribe.Endpoint
+  @api_key_auth :md5 |> :crypto.hash("secuyrity_key") |> Base.encode16()
+  @base_auth Base.encode64("username:pass")
+  @bearer_auth "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxIn0.rTCH8cLoGxAm_xw68z-zXVKi9ie6xJn9tnVWjd_9ftE"
+
+  def users_index(opts \\ []) do
+    conn = conn()
+
+    conn
+    |> put_needed_headers(opts)
```

Comment:
```suggestion
    conn()
    |> put_needed_headers(opts)
```
WDYT? Seems a bit easier to read.

Reply:
I need conn to generate the url too. So I created the var to don't build it twice.